spec/models/role_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Role, type: :model do
+  subject { FactoryGirl.create(:role) }
+
+  describe 'validations' do
+    it { is_expected. to validate_presence_of :name }
+    it { is_expected. to validate_uniqueness_of(:name).case_insensitive }
+    it { is_expected. to be_valid }
+
+    it { is_expected. to have_many :person_roles }
+    it { is_expected. to have_many(:people).through(:person_roles) }
+
+    it "expects name to be stored lower-case" do
```

Comment:
Don't want to be a penny picker but just a heads up there is some inconsistency between single and double quotes being used in here. Generally you only use double quotes when you are performing string interpolation inside the string.